docs/BigQueryMultiTable-batchsink.md

```diff
@@ -36,6 +36,11 @@ It will be automatically created if it does not exist, but will not be automatic
 Temporary data will be deleted after it is loaded into BigQuery. If it is not provided, a unique
 bucket will be created and then deleted after the run finishes.
 
+**WriteDisposition**: Type of write disposition to use. This can be set to Empty, Truncate, or Append.
```

Comment:
Can we keep it simple. Like normal English that non tech users can understand

Reply:
Hi, could you clarify which part needs to be simplified? This is currently a direct mapping to the doc here:

https://cloud.google.com/bigquery/docs/reference/auditlogs/rest/Shared.Types/WriteDisposition

Reply:
Write Disposition corresponds to the BigQuery concept, but the BigQuery concept isn't very simple. I think all we really need is a "Truncate Table" option. If true, the plugin sets write disposition to 'truncate'. Not sure we really need 'Empty'.

Reply:
That makes sense, I've gone ahead and simplified it to "Truncate Table" with a true-false radio group.

Reply:
Additionally, should we need to change the mode in the future, all we would need to do is change the widget JSON and the documentation.